data/project-templates/react-ts/src/components/NodeVersionCheckTask.tsx

```diff
@@ -0,0 +1,22 @@
+import { TaskExecutor, useTask, Worker } from "@golem-sdk/react";
+
+export function NodeVersionCheckTask({ executor }: { executor: TaskExecutor }) {
+  const { isRunning, result, run, error } = useTask<string>(executor);
+
+  const getNodeVersionTask: Worker<unknown, string> = async (ctx) => {
```

Comment:
Check if the first type parameter is dropped in the beta of golem-js

Reply:
@SewerynKras , I think this type will be modified when you will be dropping `map` and `forEach` support :)